schemas/objects.graphql

```diff
@@ -0,0 +1,34 @@
+type User {
+    id: ID!
+    displayName: String
+    email: String
+    emailVerified: String
+    photoURL: String
+    firstName: String
+    lastName: String
+    yearLevel: String
+    role: String 
+    created: DateTime 
+    modified: DateTime 
+}
+
+type Question {
+    id: ID!

```

Comment:
Let's define all the variables from the database schema.

also, define a nested query e.g

```gql
type Quiz {
	...
	questions: [Question]
	...
}
```

some of the nested query you can implement:
- Quiz -> Question
- Question -> Options
- UserQuiz -> Quiz, User (remember to remove UserAnswer)
- UserQuiz -> UserQuizAnswer

Reply:
We can also do queries inside types e.g
```gql
type Quiz {
	...
	questions: [Question]
	"Get question by index"
	question(index: Int!): Question 
	...
}
```